control_panel_api/services.py

```diff
@@ -1,6 +1,6 @@
 from django.conf import settings
 
-from . import aws
+from .aws import aws_api_client as aws
```

Comment:
I love how as the name of the methods didn't change this means this is the only change in the file 👍